File: client/src/app/request/SF0030207.req.ts
import {ProductFile} from "../model/core/ProductFile.model";
export class SF0030207Req {
    productFile: ProductFile;
    fileCode: string

    constructor(productFile: ProductFile, fileCode: string) {
        this.productFile = productFile;
        this.fileCode = fileCode;
    }

}

File: app/vn/vnext/sefuri/sf/json/request/SF0030207Req.java
package vn.vnext.sefuri.sf.json.request;

import com.fasterxml.jackson.annotation.JsonProperty;
import vn.vnext.sefuri.sf.json.common.AbstractJson;
import vn.vnext.sefuri.sf.json.core.ProductFileJson;

/**
 * Created by TungNT on 1/5/2017.
 */
public class SF0030207Req extends AbstractJson {
    @JsonProperty("productFile")
    private ProductFileJson productFileJson;

    @JsonProperty("fileCode")
    private String fileCode;

    public ProductFileJson getProductFileJson() {
        return productFileJson;
    }

    public void setProductFileJson(ProductFileJson productFileJson) {
        this.productFileJson = productFileJson;
    }

    public String getFileCode() {
        return fileCode;
    }

    public void setFileCode(String fileCode) {
        this.fileCode = fileCode;
    }
}
